source/preprocessor.tex

```diff
@@ -811,11 +811,14 @@
 are processed just as in normal text
 (i.e., each identifier currently defined as a macro name
 is replaced by its replacement list of preprocessing tokens).
+\begin{note}
 An \tcode{import} directive
 matching the first two forms of a \grammarterm{pp-import}
 instructs the preprocessor to import macros
 from the header unit\iref{module.import}
-denoted by the \grammarterm{header-name}.
+denoted by the \grammarterm{header-name},
+as described below.
+\end{note}
 \indextext{point of!macro import|see{macro, point of import}}%
 The \defnx{point of macro import}{macro!point of import} for the
 first two forms of \grammarterm{pp-import} is
```

Comment:
For completeness, should we say "The last form of *pp-import* is only considered if the first two forms did not match<ins>, and does not have a point of macro import</ins>." below?

Reply:
fixed